pkg/docker/docker.go

```diff
@@ -222,3 +229,31 @@ func ExtractResult(reader io.Reader, command []string) *Result {
 	}
 	return &result
 }
+
+func checkImageExist(ctx context.Context, cli *client.Client, image string, notag bool) bool {
+	log.Debugf("docker: checking existence of the image '%s'", image)
```

Comment:
Should be `"Docker: Checking exisitence"`

Reply:
https://blog.golang.org/error-handling-and-go
Check this^ blog out. Though the convention is defined for error statements, I have applied similar to debug statements as well to maintain the consistency.

Reply:
Have they mentioned about the case?